pyanaconda/ui/gui/__init__.py

```diff
@@ -865,6 +865,11 @@ def run(self):
             Gtk.StyleContext.add_provider_for_screen(Gdk.Screen.get_default(), provider,
                     Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION)
 
+            # Add the application icons to the theme
+            icon_path = os.environ.get("ANACONDA_DATA", "/usr/share/anaconda/pixmaps")
```

Comment:
I think this needs to be ``os.path.join(ANACONDA_DATA, "pixmaps")`` similarly to how ``css_path`` is constructed above.